README.md

```
@@ -27,6 +28,38 @@ This project extends [Pipecat's WebSocket server implementation](https://github.
+- WebSocket server for real-time communication
+- Designed to be used as a backend service behind a more comprehensive API
+- No authentication (meant to be handled by the parent API)
+
+#### API Endpoints
+
+1. Root endpoint (`GET /`):
+   - Health check endpoint
+   - Returns: `{"message": "MeetingBaas Bot API is running"}`
+
+2. Run Bots (`POST /run-bots`):
+   ```json
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines
null

(MD031, blanks-around-fences)